src/test-helpers/createTestApp.tsx

```diff
@@ -80,4 +79,13 @@ export const refreshTestApp = async () => {
   await act(vi.runOnlyPendingTimersAsync)
 }
 
+/** Clear exisiting event listeners, but without clearing the app. */
+export const cleanupTestEventHandlers = async () => {
```

Comment:
had to add this, as I really want not to clear the whole app between the test runs of the tutorial